app/views/teams/_form.html.slim

```diff
@@ -77,6 +77,7 @@
       = f.simple_fields_for :conference_preferences, f.object.conference_preferences.sort{ |a,b| a.option <=> b.option} do |c|
         = c.hidden_field :option, value: c.object.option
         = c.input :conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: c.object.option == 1 ? "Primary choice" : "Secondary Choice", prompt: "Conference", wrapper_html: { class: 'col-md-8' }
+    = link_to 'I want to suggest another conference!', new_conference_path(team: @team.id), class: 'btn btn-primary col-sm-offset-3'
```

Comment:
Furthermore, this link will discard whatever has been entered into the form. It should either open in a new tab or add a JS confirmation popup before it follows the link.  The latter would be preferable since a newly created conference would not show up in the open form. The page needs to be reloaded anyway to repopulate the select box.